deployment/ansible/roles/driver.database/templates/monit-postgresql.cfg.j2

```diff
@@ -0,0 +1,3 @@
+check process postgresql with pidfile "/var/run/postgresql/9.4-main.pid"
```

Comment:
Probably want to use `{{ postgresql_version }}` here to make future version upgrades less error-prone.